<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_test">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_headline">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>900</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Haushaltsbuch</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>200</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="einkommen_lbl">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Einkommen</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="einkommen_date">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="einkommen_kat">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Kategorie Auswahl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gehalt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vermietung</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aktien, ETF, ...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Schenkung</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bafög</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sonstige</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="einkommen_hoehe">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Höhe</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_einkommen">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Hinzufügen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>200</width>
      <height>222</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="ausgaben_lbl">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Ausgaben</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="ausgaben_date">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2022</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ausgaben_kat">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Kategorie Auswahl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lebensmittel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Elektrogeräte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Miete (Warm)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Konsumgüter / Luxusgüter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Transportkosten</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sonstige Ausgaben</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Versicherungen</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ausgaben_artikel">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Artikel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ausgaben_preis">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Preis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ausgaben_mwst">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>MwSt Auswahl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0 %</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7 %</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19 %</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_ausgaben">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Hinzufügen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>einkommen_date</tabstop>
  <tabstop>einkommen_kat</tabstop>
  <tabstop>einkommen_hoehe</tabstop>
  <tabstop>ausgaben_date</tabstop>
  <tabstop>ausgaben_kat</tabstop>
  <tabstop>ausgaben_artikel</tabstop>
  <tabstop>ausgaben_preis</tabstop>
  <tabstop>ausgaben_mwst</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
